libraries/botbuilder-core/botbuilder/core/activity_handler.py

```diff
@@ -344,3 +351,56 @@ class ActivityHandler:
             conversation update, message reaction, or event activity, it calls this method.
         """
         return
+
+    async def on_invoke_activity(  # pylint: disable=unused-argument
+        self, turn_context: TurnContext
+    ):
+        """
+        Registers an activity event handler for the _invoke_ event, emitted for every incoming event activity.
+
+        :param turn_context: The context object for this turn
+        :type turn_context: :class:`botbuilder.core.TurnContext`
+
+        :returns: A task that represents the work queued to execute
+        """
+
+        if (
+            turn_context.activity.name == SignInConstants.verify_state_operation_name
+            or turn_context.activity.name
+            == SignInConstants.token_exchange_operation_name
```

Comment:
Why line 370 is not at the end of 369? We all have wide screen monitors, doesn't seem useful to limit to 70 columns

Reply:
The python SDK follows a formatting style called black (https://black.readthedocs.io/en/stable/) and checks the formatting in every build.

The formatting is done automatically.

Reply:
pd. I will try to do it 1 line and check if the formatter doesn't complain